proxy/src/coco.rs

```diff
@@ -88,20 +90,24 @@ impl Peer {
         clippy::wildcard_enum_match_arm,
         clippy::match_wildcard_for_single_variants
     )]
-    pub fn list_projects(&self) -> Result<Vec<project::Project<entity::Draft>>, error::Error> {
+    pub fn list_projects(&self) -> Result<Vec<Project>, error::Error> {
         self.with_api(|api| {
             let storage = api.storage();
-            Ok(storage
-                .all_metadata()?
-                .flat_map(|entity| {
-                    entity.ok()?.try_map(|info| match info {
-                        entity::data::EntityInfo::Project(info) => Some(info),
-                        _ => None,
+            let project_meta = storage.all_metadata()?.flat_map(|entity| {
+                entity.ok()?.try_map(|info| match info {
+                    entity::data::EntityInfo::Project(info) => Some(info),
+                    _ => None,
+                })
+            });
+            project_meta
+                .map(|project| {
+                    Self::with_browser(api, &project.urn(), |browser| {
+                        let stats = browser.get_stats()?;
+                        Ok(Project::from_project_stats(project, stats))
                     })
                 })
-                .collect())
+                .collect::<Result<Vec<Project>, error::Error>>()
```

Comment:
I think this turbo fish (`::<>`) type declaration can go

Reply:
![turbofish](https://user-images.githubusercontent.com/2712962/85905832-61774900-b7c1-11ea-8eb3-3f3d288d2ad9.png)

i will vanquish him